CHANGELOG.md

```diff
@@ -1,14 +1,122 @@
 # CHANGELOG
 
+# Unreleased changes
+
+_None outstanding_
```

Comment:
Do we need this top section?

Reply:
We don't right now. I was putting it there as a prompt for us to build this up incrementally between future releases.

Do you dislike it being there?

Reply:
👍